docs/_layouts/default.html

```diff
@@ -26,7 +26,7 @@
 		<script src="https://d3js.org/d3.v4.min.js"></script>
 		<script src="https://momentjs.com/downloads/moment-with-locales.min.js"></script>
 		<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.0/Chart.min.js"></script>
-		<script src="{{ site.baseurl }}/assets/js/charts.js?version=62c28e5"></script>
+		<script src="{{ site.baseurl }}/assets/js/charts.js?version=1ff0187"></script>
```

Comment:
is this a version update? let's not mix version updates with normal changes in the future ;-)

Reply:
No, that’s a resource tag.
You use these to force browsers to reload resources, even with browser caching enabled.